### DISCUSSION TITLE
How do you the galaxy brain achievement?

### DISCUSSION THREAD

**Discussion by sindrimt** [community/community#18384] [Other Feature Feedback, Questions, & Ideas] [answered]
Does anyone know how to get the galaxy brain achievement?

**Comment by GIHAA** [comment]
I think you have to get a question marked as answered from the author

**Comment by regidf12** [comment]
Pull Shark — за созданный и принятый пул-реквест;

YOLO — за самостоятельное слияние веток в проекте;

Galaxy Brain — за полезный ответ на вопрос на GitHub Discussions.

**Comment by hulchenko** [comment]
Info in this repo might be useful for you @sindrimt:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by melvincwng** [comment]
Hi @sindrimt, to get the 'Galaxy Brain' achievement, you need to have 2 answered discussions.

For any question in the 'Discussions' tab, there should be a 'Mark as answer' button. When you have 2 answers marked, you should unlock the badge.

Hope this clarifies.

Reference: https://github.com/Schweinepriester/github-profile-achievements

**Comment by saltbo** [comment, marked as answer]
Hi there! The Galaxy Brain Achievement requires the qualifying Discussions to be marked as answers by another user that's not the author.

**Comment by rareyesrd** [comment]
Hello! 

The Galaxy Brain Achievement requires (two) qualifying Discussions to be marked as answers.

**Comment by gabrielvlima** [comment]
Hi, the Galaxy Brain, needs 2 accepted answers.

**Comment by jafludo** [comment]
Needs 2 accepted answers.

**Comment by rimsha-salahuddin** [comment]
Just go to the GitHub community page and answers the questions (doubts) of others posted under the Discussions section and your answer should be accepted by GitHub. When you will answer atleast 2 questions then you will achieve Galaxy Brain.

**Comment by IvailoPetrov06** [comment]
@sindrimt You need a meaningful and true question in a discussion on GitHub.

**Comment by PaulThomas20002** [comment]
You need to become the marked answer to 2 discussions
but it seems like there are more conditions, for example you don't seem to be able to mark yourself as the answer.

But I recommend you to select this as the answer, because then I should have answered 2 discussions and be marked by the OP as the answer, so that should earn Galaxy Brain.

Thanks for reading my answer

**Comment by zroussama** [comment]
To achieve the "**Galaxy Brain**" badge on GitHub, you need to meet the following criteria:

- **Answer Two Discussions:** Provide meaningful answers to at least two discussions in GitHub's public repositories.

- **Accepted Answers:** Your answers must be marked as "accepted" by the original poster or someone with appropriate permissions. Self-marked answers do not count to prevent abuse.

- **Public Repositories:** Ensure that the discussions are part of public repositories, as private ones do not qualify for this achievement.

As of February 2024, **achievements are no longer available in GitHub Community Discussions** due to high volumes of low-quality posts. However, you can still earn this badge in other repository discussions outside of the main GitHub Community forum[1](https://github.com/orgs/community/discussions/18384)[2](https://github.com/orgs/community/discussions/143321)[5](https://github.com/orgs/community/discussions/18796).
